packages/base/src/dialogs/symbology/symbologyUtils.ts

```diff
@@ -76,10 +76,16 @@ export namespace VectorUtils {
 
     const stopOutputPairs: IStopRow[] = [];
 
-    for (let i = 3; i < color['circle-radius'].length; i += 2) {
+    const circleRadius = color['circle-radius'];
+
+    if (!Array.isArray(circleRadius) || circleRadius.length < 4) {
```

Comment:
Can you explain the rationale for this change? Is this the cause of the bug when changing render types?